app/src/main/java/com/gaurav/avnc/util/AppPreferences.kt

```diff
@@ -54,6 +54,7 @@ class AppPreferences(context: Context) {
         val vkShowAll; get() = prefs.getBoolean("vk_show_all", false)
 
         val mousePassthrough; get() = prefs.getBoolean("mouse_passthrough", true)
+        val hideLocalCursor; get() = prefs.getBoolean("hide_local_cursor", true)
```

Comment:
```suggestion
        val hideLocalCursor; get() = prefs.getBoolean("hide_local_cursor", false)
```

Please change the default value for this preference to `false`. 
- Existing users (like me), who use the 'Hide Remote Cursor' server option, suddenly won't *any* cursor at all, if this pref is on by default.
- This behavior of hiding cursor is unexpected from user POV, and it interacts badly with the toolbar (try opening the toolbar and using it). Hence it should be opt-in.
- It breaks the experience of users who disable 'Mouse Passthrough' (probably not many, but still).